CONTRIBUTING.md

```diff
@@ -88,8 +87,44 @@ Follow these steps to update a locale for the SAS Extension for VSCode:
 
 ### Locale contributors
 
-| Language                 | VSCode Language ID | Contributor |
-| ------------------------ | ------------------ | ----------- |
-| **German**               | de                 | David Weik  |
-| **Chinese (Simplified)** | zh-cn              | Wei Wu      |
-| **Portuguese (Brazil)**  | pt-br              | Mark Jordan |
+| Language                 | VSCode Language ID | Contributor     |
+| ------------------------ | ------------------ | --------------- |
+| **French**               | fr                 | Vincent Rejany  |
+| **Italian**              | it                 | Lorenzo Roccato |
+| **German**               | de                 | David Weik      |
+| **Chinese (Simplified)** | zh-cn              | Wei Wu          |
+| **Japanese**             | ja                 | Masayuki Iizuka |
+| **Korean**               | ko                 | Meilan Ji       |
+| **Portuguese (Brazil)**  | pt-br              | Mark Jordan     |
+
+## Run single test file
+
+- Open the `.test.ts` file you want to run
+- Switch to the `Run and Debug` view in the VS Code Activity Bar (Ctrl+Shift+D).
+- Select `Language Server E2E Test` from the drop down.
+- Press ▷ to run the launch config (F5).
+- See test result in the `Debug Console` panel.
+
+# Testing unpublished versions
+
+You can still try out any commit or pull request if you don't want to manually build from source code.
+
+## Download VSIX file
+
+- Open below page with your browser
+  - For main branch https://github.com/sassoftware/vscode-sas-extension/actions/workflows/package.yml
+  - For Pull Request https://github.com/sassoftware/vscode-sas-extension/actions/workflows/pr.yml
+- Select the commit/PR you want.
+- Download the `artifact.zip` file from the Artifacts section.
+- Unzip it to get the VSIX file.
+
+## Install VSIX file
+
+- Open the `Extensions` view on the VS Code Activity Bar.
+- Click the `...` from the top right of the Extensions pane, and select `Install from VSIX...`, select the downloaded VSIX file.
+- Restart VS Code
+
+**Note**:
+
+- When testing VSIX files, it's usually a good idea to turn off "Extensions: Auto Update" in your VS Code settings to prevent it auto-updated to published version.
+- When switching between multiple VSIX files, it's usually a good idea to clean up the [installation directory](https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed) after uninstall previous one. Otherwise VS Code may cache it as un-published versions may look same.
```

Comment:
```suggestion
- When switching between multiple VSIX files, it's usually a good idea to clean up the [installation directory](https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed) after uninstalling a previous version. Otherwise VS Code may cache it as un-published versions may look same.
```